Doc/biopdb_faq.tex

```diff
@@ -71,7 +71,7 @@
 {\normalsize Institute of Molecular Biology}\\
 {\normalsize University of Copenhagen }\\
 {\normalsize Universitetsparken 15, Bygning 10}\\
-{\normalsize DK-2100 K�benhavn �}\\
+{\normalsize DK-2100 K�benhavn �}\\
```

Comment:
Something weird has happened to the special characters. I've changed this on the master to avoid the problem: 1891973fe3cf5a4ad7e05f6a9ccd8d5d84f48a44